### DISCUSSION TITLE
Does the latest gef-classic version require JRE 11?

### DISCUSSION THREAD

**Discussion by QGMeta** [eclipse-gef/gef-classic#97] [Q&A] [answered]
Does the latest gef-classic version require JRE 11?

**Comment by QGMeta** [comment, marked as answer]
Where can I obtain the P2 update site zip of the release version?

**Reply by azoitl** [reply]
Unfortunately we don't have a zip of the update site yet. I'm working on it. In the meantime you can get the latest version here: https://download.eclipse.org/tools/gef/classic/releases/latest/

**Reply by QGMeta** [reply]
thank u for reply, looking forward.

**Comment by azoitl** [comment]
> Does the latest gef-classic version require JRE 11?
Yes like the Eclipse Platform GEF Classic now requers JRE 11.
